City-of-Helsinki/drupal-employment-services
# THF-559: THF-559: next-drupal lupdate 1.6

updated Next-drupal module

-test with branch https://github.com/City-of-Helsinki/employment-services-ui/pull/344

Testing:
- `make shell;`
- `composer install`
- `drush updb -y; drush cim -y; drush cr;`
- Create at least 1 article, 1 Basic page and 1 Landing page
- Add all the pages all components.
- See that there is no errors
